components/questionCard.js

```diff
@@ -0,0 +1,189 @@
+import React, { useState } from 'react'
+import styled from 'styled-components'
+import Toggle from 'react-toggle'
+import TextField from './TextField'
+import { COLOR, QUESTION_TYPES } from '../constants'
+import QuestionDropdown from './questionDropdown'
+import Icon from './Icon'
+
+const QuestionTitle = styled.p`
+  color: ${COLOR.MIDNIGHT_PURPLE_DEEP};
+  font-weight: 800;
+  font-size: 16px;
+  margin: 10px 0px;
+`
+
+const OptionsContent = styled.div`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  gap: 5px;
+`
+
+const StyledQuestionButton = styled.button`
+  border: none;
+  background: transparent;
+  display: block;
+  margin: 10px auto;
+  &:hover {
+    transform: scale(1.2);
+  }
+  i {
+    transform: scale(1.3);
+  }
+`
+
+const TitleBar = styled.div`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+`
+
+const StyledField = styled.input`
+  color: ${COLOR.MIDNIGHT_PURPLE_DEEP};
+  border: none;
+  background-color: none;
+  outline: 0;
+  font-size: 24px;
+  font-weight: 700;
+  width: 70%;
+  height: 30px;
+  overflow: auto;
+  border: none;
+  font-family: 'HK Grotesk';
+  cursor: none;
+  &:focus {
+    border: none;
+    background-color: none;
+    outline: 0;
+  }
+`
+
+const Bar = styled.div`
+  display: flex;
+  gap: 10px;
+  align-items: center;
+  i {
+    transform: scale(1.3);
+  }
+`
+
+const QuestionCard = ({ question, removeQuestion, id, moveUp, moveDown, handleChange, duplicateQuestion }) => {
+  // const [required, setRequired] = useState(question.required || false)
+  const [isToggled, setToggled] = useState(question.toggled || true)
+
+  const handleOptionChange = (index, value) => {
+    const newOptions = [...question.options]
+    newOptions[index] = value
+    handleChange(id, 'options', newOptions)
+  }
+
+  const addOption = () => {
+    handleChange(id, 'options', [...question.options, ''])
+  }
+
+  const removeOption = index => {
+    const newOptions = question.options.filter((_, i) => i !== index)
+    handleChange(id, 'options', newOptions)
+  }
+
+  return (
+    <div
+      style={{
+        border: `1px solid ${COLOR.MIDNIGHT_PURPLE_DEEP}`,
+        borderRadius: '16px',
+        padding: '30px',
+        width: '40vw',
+      }}
+    >
+      <TitleBar>
+        <Bar style={{ width: '90%' }}>
+          <Icon color={COLOR.MIDNIGHT_PURPLE_DEEP} icon="question-circle" />
+          <StyledField type="text" placeholder="Untitled" value={question.title} readonly />
```

Comment:
i think it should be `readOnly`! you can also probably remove `cursor: none` on line 55 to make the text selectable

Reply:
Can we use styled components for the inline css + div?

+1, when you click on the title bar, a cursor pops up, but it's not editable, can be confusing to users. can we remove the cursor and make it unclickable?

![image](https://github.com/user-attachments/assets/f47848c4-6e85-4e2b-bac5-8c6a72ebcf14)